build/docker/ubuntu-trusty/Dockerfile

```diff
@@ -217,6 +217,62 @@ RUN rm -rf /var/cache/apt/* && \
     rm -rf /tmp/* && \
     rm -rf /var/tmp/*
 
+# Ruby
```

Comment:
This change looks incorrect.  It duplicates other parts of the file.

Reply:
It's because I messed up the rebase. Will fix.

Reply:
Fixed.